src/wait-for-element-to-be-removed.js

```diff
@@ -12,13 +12,28 @@ function initialCheck(elements) {
   }
 }
 
+function wrapFunctionCallback(callback) {
```

Comment:
"wrap" it with what? It would help a lot if we could come up with a less generic name for this function.